frontend-react/src/contexts/SessionContext.tsx

```diff
@@ -32,12 +37,17 @@ const SessionProvider = ({
 }: React.PropsWithChildren<ISessionProviderProps>) => {
     const { authState } = oktaHook();
 
-    const membershipController = useOktaMemberships(authState);
+    const {
+        state: { memberships, activeMembership },
+        dispatch,
+    } = useOktaMemberships(authState);
     return (
         <SessionContext.Provider
             value={{
                 oktaToken: authState?.accessToken,
-                memberships: membershipController,
+                memberships,
```

Comment:
almost all of the changes in this PR are downstream consequences of changing the signature of this provider value